code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm

```diff
@@ -813,7 +678,176 @@
 	M.reagents.remove_reagent("sugar", 3)
 	..()
 	return
+	
+//NamePendingChem Chems
+
+/datum/reagent/medicine/sleeptoxin //Old Morphine
+	name = "Sleep Toxin"
+	id = "sleeptoxin"
+	description = "A safe way to send a patient to sleep. 8 units is required to send them to sleep."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+	metabolization_rate = REAGENTS_METABOLISM
+
+/datum/reagent/medicine/sleeptoxin/on_mob_life(mob/living/carbon/human/M)
+	if(current_cycle == 7)
+		M << "<span class='warning'>You start to feel tired...</span>" //Warning when the victim is starting to pass out
+	if(current_cycle > 15)
```

Comment:
Description says 8 units is required but does this mean after 15 units they fall asleep?